### PR TITLE
[devtools] polish breakpoints doc

### PR DESCRIPTION
**Pull request by kaycebasques**
* Updates screenshots to use the new standard.
* Uses consistent, general step-by-step instructions on how to set each type of breakpoint.
* Adds some overview information to top of doc explaining when to use each type.
* Updates link text and URLs in other docs.
* Changes the DOM change and exception breakpoint sections back to the standard format for guide sections. For a while I was experimenting with an interactive tutorial format for each of these sections. These are largely unnecessary now, now that the Get Started Debugging JS guide is shipped. It was also weird to have sections that were pretty much tutorials within a guide.
* Changes title of doc to "Breakpoints Guide". Eventually, all the DevTools docs will follow this format.
* Renames URL to `breakpoints` from `add-breakpoints` and redirects the old to the new. Kinda unnecessary, but I really like clean URLs

### CODE REVIEW

**Comment by kaycebasques** [comment]
@petele or @jpmedley please review at your leisure

File: src/content/en/tools/chrome-devtools/_toc.yaml [hunk 1]
```diff
@@ -53,8 +53,8 @@ toc:
       section:
       - title: Get Started
         path: /web/tools/chrome-devtools/javascript/
-      - title: Set Breakpoints
-        path: /web/tools/chrome-devtools/javascript/add-breakpoints
+      - title: Breakpoints Guide
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/_toc.yaml]: Is there any particular reason you're moving away from a title beginning with a verb? After reading the article, I think 'Set Breakpoints' is more appropriate.

File: src/content/en/tools/chrome-devtools/javascript/breakpoints.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+## Overview of when to use each breakpoint type {: #overview }
+
+The most well-known type of breakpoint is line-of-code. But line-of-code
+breakpoints can be inefficient to set, especially if you don't know exactly
+where to look, or if you are working with a large codebase. You can save
+yourself time when debugging by knowing how and when to use the other types
+of breakpoints.
+
+<table>
+  <tr><th>Breakpoint Type</th><th>Use This Type When You...</th></tr>
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/javascript/breakpoints.md]: Maybe change 'Use This Type When You...' to 'Use This When You'.

File: src/content/en/tools/chrome-devtools/javascript/breakpoints.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+    <td><a href="#dom">DOM</a></td>
+    <td>
+      Want to pause on the code that changes or removes a specific DOM
+      node, or its children.
+    </td>
+  </tr>
+  <tr>
+    <td><a href="#xhr">XHR</a></td>
+    <td>
+      Want to pause when an XHR URL contains a string pattern.
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/javascript/breakpoints.md]: Does this work on fetch yet? Might want to say something about this either way.

File: src/content/en/tools/chrome-devtools/javascript/breakpoints.md [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+  </figcaption>
+</figure>
+
+### Manage line-of-code breakpoints {: #manage-loc }
+
+Use the **Breakpoints** pane to disable or remove line-of-code breakpoints from
+a single location.
+
+* Check the checkbox next to an entry to disable that breakpoint.
+* Right-click over an entry to remove that breakpoint.
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/javascript/breakpoints.md]: Delete 'over'.

File: src/content/en/tools/chrome-devtools/javascript/breakpoints.md [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+* Right-click anywhere in the **Breakpoints** pane to deactivate all
+  breakpoints, disable all breakpoints, or remove all breakpoints. Disabling
+  all breakpoints is equivalent to unchecking each one. Deactivating all
+  breakpoints instructs DevTools to ignore all line-of-code breakpoints, but to
+  also maintain preserve their enabled state so that they are in the same
+  state as before when you re-activate them.
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/javascript/breakpoints.md]: How about 'same place as before' instead of 'same state as before'?

Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/javascript/breakpoints.md]: I had to go old-school to confirm. I.e., I had to look in my actual physical copy of Webster's. 'Reactivate' (no hyphen) is the proper form.

File: src/content/en/tools/chrome-devtools/javascript/breakpoints.md [hunk 6]
```diff
@@ -0,0 +1,238 @@
...
+
+<figure>
+  <img src="imgs/breakpoints-pane.png"
+       alt="The Breakpoints pane."
+  <figcaption>
+    <b>Figure 3</b>: The <b>Breakpoints</b> pane shows that there are two
+    line-of-code breakpoints: one on line 15 of <code>get-started.js</code>,
+    and another on line 32.
+  </figcaption>
+</figure>
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/javascript/breakpoints.md]: The context for this is about deactivating and reactivating breakpoints. I think you should (1) use an image with translucent (deactivated) breakpoints and (2) call out that fact in the caption.

File: src/content/en/tools/chrome-devtools/javascript/breakpoints.md [hunk 7]
```diff
@@ -0,0 +1,238 @@
...
+## DOM change breakpoints {: #dom }
+
+Use a DOM change breakpoint when you want to pause on the code that changes
+a DOM node or its children.
+
+To set a DOM change breakpoint:
+
+1. Click the **Elements** tab.
+1. Go the element that you want to set the breakpoint on.
+1. Right-click on the element.
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/javascript/breakpoints.md]: Change to 'Right-click the element'. ('On' removed.)

File: src/content/en/tools/chrome-devtools/javascript/breakpoints.md [hunk 8]
```diff
@@ -0,0 +1,238 @@
...
+1. Hover over **Break on** then select **Subtree modifications**, **Attribute
+  modifications**, or **Node removal**.
+
+<figure>
+  <img src="imgs/dom-change-breakpoint.png"
+       alt="The context menu for creating a DOM change breakpoint."
+  <figcaption>
+    <b>Figure 4</b>: The context menu for creating a DOM change breakpoint.
+  </figcaption>
+</figure>
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/javascript/breakpoints.md]: Check throughout. Figure captions don't get periods except in [specific circumstances](https://developers.google.com/style/images#text-associated-with-images).

File: src/content/en/tools/chrome-devtools/javascript/breakpoints.md [hunk 9]
```diff
@@ -0,0 +1,238 @@
...
+
+### More information on each type of DOM change breakpoint
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/javascript/breakpoints.md]: Change to 'More about DOM change breakpoints'. Or maybe, 'Types of DOM change breakpoints'. Shorter is better for the page toc.

File: src/content/en/tools/chrome-devtools/javascript/breakpoints.md [hunk 10]
```diff
@@ -0,0 +1,238 @@
...
+  node is removed or added, or the contents of a child are changed. Not
+  triggered on child node attribute changes, or on any changes to the
+  currently-selected node.
+
+* **Attributes modifications**: Triggered when an attribute is added or removed
+  on the currently-selected node, or when an attribute value changes.
+
+* **Node Removal**: Triggered when the currently-selected node is removed.
+
+## XHR breakpoints {: #xhr }
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/javascript/breakpoints.md]: Again, what about fetch? (Maybe there's a place for setting a conditional breakpoint inside a fetch handler.)

File: src/content/en/tools/chrome-devtools/javascript/breakpoints.md [hunk 11]
```diff
@@ -0,0 +1,238 @@
...
+
+Use exception breakpoints when you want to pause on the line of code that's
+throwing a caught or uncaught exception.
+
+1. Click the **Sources** tab.
+1. Click **Pause on exceptions** ![Pause on
+   exceptions](imgs/pause-on-exceptions.png){:.devtools-inline}. It turns blue
+   when enabled.
+1. (Optional) Check the **Pause On Caught Exceptions** checkbox if you also
+   want to pause on caught exceptions, in addition to uncaught ones.
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/javascript/breakpoints.md]: There's no figure for this.

File: src/content/en/tools/chrome-devtools/_toc.yaml (hunk 1 above)
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/_toc.yaml]: Good question. My reasoning is that, with the current task-based titles, you don't know what type of doc you're getting. It's annoying to get a reference when you were hoping for a tutorial. However, this new approach could be a mistake, for all the reasons that task-based names are more helpful.

Also, the directories are still task-based. E.g. "Breakpoints Guide" is nested in "Inspect and Debug JavaScript". But that's not helpful if you're coming from Search.

Style Guide doesn't give recommendations one way or the other on doc titles.

Either way, a better task-based title would be "Pause Your Code With Breakpoints". "Set Breakpoints" and "Breakpoints Guide" are both feature-based: neither's useful if you don't know what a breakpoint is.

Let's leave this for now but I can revisit in a later PR

File: src/content/en/tools/chrome-devtools/javascript/breakpoints.md (hunk 2 above)
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/javascript/breakpoints.md]: Done

File: src/content/en/tools/chrome-devtools/javascript/breakpoints.md (hunk 3 above)
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/javascript/breakpoints.md]: Good catch! Yes, it does.

Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/javascript/breakpoints.md]: Done

File: src/content/en/tools/chrome-devtools/javascript/breakpoints.md (hunk 4 above)
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/javascript/breakpoints.md]: Sounds so weird to me... but done.

File: src/content/en/tools/chrome-devtools/javascript/breakpoints.md (hunk 5 above)
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/javascript/breakpoints.md]: State is more accurate. Gonna leave this one.

Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/javascript/breakpoints.md]: Done

File: src/content/en/tools/chrome-devtools/javascript/breakpoints.md (hunk 7 above)
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/javascript/breakpoints.md]: Done

File: src/content/en/tools/chrome-devtools/javascript/breakpoints.md (hunk 9 above)
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/javascript/breakpoints.md]: Nice, thanks. Done

File: src/content/en/tools/chrome-devtools/javascript/breakpoints.md (hunk 10 above)
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/javascript/breakpoints.md]: Done

File: src/content/en/tools/chrome-devtools/javascript/breakpoints.md (hunk 8 above)
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/javascript/breakpoints.md]: Bah, I didn't know that rule. Thanks. Done

File: src/content/en/tools/chrome-devtools/javascript/breakpoints.md (hunk 11 above)
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/javascript/breakpoints.md]: Done

**Comment by kaycebasques** [comment]
OK @jpmedley... I updated per yer review

File: src/content/en/tools/chrome-devtools/_toc.yaml (hunk 1 above)
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/_toc.yaml]: Per in-person debate I decided to change to "Pause Your Code With Breakpoints"

**Pull request merged**